cpu/cortexm_common/ldscripts/cortexm_base.ld

```diff
@@ -113,6 +113,9 @@ SECTIONS
         KEEP (*(.isr_stack))
         . = ALIGN(8);
         _estack = .;
+        KEEP (*(.puf_stack))
+        . = ALIGN(8);
```

Comment:
you seem to pad 8 bytes here in the linker, but only use 4 bytes later. Shouldn't we use rather `ALIGN(4)` then?